model/registry/messaging.yaml

```diff
@@ -205,4 +205,4 @@ groups:
       - id: system
         type: string
         brief: 'A string identifying the messaging system.'
-        examples: ['kafka', 'rabbitmq', 'rocketmq', 'activemq', 'AmazonSQS']
```

Comment:
Why was AmazonSQS removed? And why there isn't `gcp_pubsub` here? 🤔

Reply:
I was proposing to remove proprietary vendors from this list. Also to make clear that it's an example, and doesn't need to be an exhaustive list of possible values: https://github.com/open-telemetry/semantic-conventions/pull/490#discussion_r1386918001

We can add them again if, but it seems to me 6 examples are quite a lot.

Reply:
I don't have a strong opinion either way. Let me know you think and I'll change it.

It does seem like the Databases page doesn't have an exhaustive list of examples under `db.system`(https://opentelemetry.io/docs/specs/semconv/database/database-spans/), but has a separate table with the values that includes proprietary vendors. Maybe we should do something like that? They have more possible values though

Reply:
Ah I see, I missed your comment @pyohannes  because it was resolved. All good!

Reply:
I think we should document the known values to be used for the different messaging systems, somewhere (to avoid spelling).
I think best is as part of tech-specific semantic conventions (similar as it's being done with `db.system` and as already started for `kafka`, `rabbitmq` and `rocketmq`).

But agree, that an exhaustive list of examples here is not necessary.

Nit: for consistency, I'd also remove `activemq` from the list of examples as we do not have activemq-specific semantic conventions, yet (or add those semantic convents).